v3/src/components/graph/components/graph.tsx

```diff
@@ -102,21 +105,28 @@ export const Graph = observer((
     }, [dotsRef, graphController])
 
     // MouseOver events, if over an element, brings up hover text
-    function showDataTip(event: MouseEvent, d: any) {
+    function showDataTip(event: MouseEvent) {
       const
         target = select(event.target as SVGSVGElement)
       if (target.node()?.nodeName === 'circle' && dataset) {
+        target.transition().duration(transitionDuration).attr('r', hoverPointRadius)
         const [, caseID] = target.property('id').split("_"),
           attrIDs = graphModel.config.uniqueTipAttributes,
           tipText = getPointTipText(dataset, caseID, attrIDs)
-        dataTip.show(tipText, event.target)
+        tipText !== '' && dataTip.show(tipText, event.target)
```

Comment:
Assuming `tipText` can't be another falsy value (like numeric `0`), you could also use:
```suggestion
        tipText && dataTip.show(tipText, event.target)
```

Reply:
But tipText can be '', in which case we don't want to show the dataTip because actually displays a small white area. (The call to getPointTipText guarantees that it is a string.)

Reply:
Exactly -- my proposed change should have the same effect with less code.